File: exercicios/1017/gc.java
import java.io.IOException;
import java.util.Scanner;


public class Main {
    
    public static void main(String[] args) throws IOException {
        double v, th, ql;
        Scanner input = new Scanner(System.in);
        th = input.nextDouble();
        v = input.nextDouble();                
        ql = (th * v) / 12.0;
        System.out.printf("%.3f\n", ql);
        input.close();   
    }    
}

File: exercicios/1017/gc.kt
import java.util.Scanner

fun main( args: Array<String> ){
    val input = Scanner(System.`in`)
    var v : Double  = input.nextDouble()
    var th : Double  = input.nextDouble()
    val ql = (v * th) / 12.0
    println("%.3f".format(ql))
}